transports/websocket-transport/src/webSocketTransport.ts

```diff
@@ -134,6 +152,7 @@ export class WebSocketTransport extends Transport {
   }
 
   initializeWebsocket(authBundle: any): ReconnectingWebSocket {
+    console.log("Initializing websocket", authBundle);
```

Comment:
nit: remove before merging or change to a debug log

Reply:
removed!